api/handlers/listhandler.py

```diff
@@ -61,6 +62,13 @@ def initialize_list_configurations():
             'storage_schema_file': 'note.json',
             'input_schema_file': 'note.json'
         },
+        'analyses': {
+            'storage': liststorage.AnalysesStorage,
+            'permchecker': listauth.default_sublist,
+            'use_object_id': True,
+            'storage_schema_file': 'analysis.json',
+            'input_schema_file': 'analysis.json'
```

Comment:
I don't see this schema file in the PR, maybe it was left behind in the split? Also, if you have time, it would be nice to update the Data Model v2 wiki and add the Analysis object there.